apps/src/templates/teacherDashboard/shapes.jsx

```diff
@@ -42,4 +42,4 @@ export const loadErrorShape = PropTypes.shape({
   message: PropTypes.string.isRequired,
 });
 
-export const OAuthSectionTypes = makeEnum('google_classroom', 'clever');
+export const OAuthSectionTypes = makeEnum('google_classroom', 'clever', 'microsoft_classroom');
```

Comment:
It was on the spec I built to; I figured it's easy to leave in if Microsoft Classroom is coming soon and will 'just work' when it's hooked up.